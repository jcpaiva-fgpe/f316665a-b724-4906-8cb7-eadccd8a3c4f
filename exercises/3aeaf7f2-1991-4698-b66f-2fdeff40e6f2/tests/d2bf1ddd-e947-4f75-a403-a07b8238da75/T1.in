5
T13
O1
P7
A5
S10
