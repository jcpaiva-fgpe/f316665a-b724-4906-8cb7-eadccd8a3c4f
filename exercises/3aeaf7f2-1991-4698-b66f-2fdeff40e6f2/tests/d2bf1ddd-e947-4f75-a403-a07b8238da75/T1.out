TTTTTTTTTTTTT
O
PPPPPPP
AAAAA
SSSSSSSSSS
